proposals/evaluation/uc-0-effective-acr.md

```diff
@@ -0,0 +1,56 @@
+# Effective Access Control Resource Discovery Use Cases
+
+This is part 0 of the [implementation specific use cases comparison](./use-cases.md).
+
+The resource governing access over another resource is called its effective access control resource (ACR).
+
+## 1. Discovering a resource's effective access control
+
+A Solid client needs to discover effective ACR in order to understand and or edit access permissions.
+
+Both WAC and ACP use the same effective ACR discovery mechanism.
+
+### Setup
+
+We have the following hierarchy of resources:
+
+```
+</x>
+</x.acr>
+```
+
+### Universal effective ACR discovery
+
+An agent making a GET or HEAD HTTP request on `</x>` will receive a `Link: </x.acr>; rel="acl"` header in the response that points to the above `</x.acr>`.
+
+The Link header with relationship type `acl` indicates the access control resource of a resource.
+
+### ACP
+
+In ACP, every resource has exactly 1 effective access control resource directly associated with it, and every access control resource directly governs access over itself and exactly one other resource.
+
+In ACP, access control statements can be spread over several resources; that is, an access control resource can reference other resources.
+
+In ACP, the access control system in place (that is, ACP, as opposed to WAC, for example) is indicated via a Link header of `rel="type"` `<http://www.w3.org/ns/solid/acp#AccessControlResource>`.
```

Comment:
Is that link present in response to ACR?

Reply:
Yes.